addons/pos_self_order/models/pos_config.py

```diff
@@ -250,3 +367,56 @@ def _get_default_qr_codes(
                 * number,
             }
         ]
+
+    def _compute_self_order_kiosk_url(self):
+        for record in self:
+            domain = self.get_base_url()
+            record.self_order_kiosk_url = '%s/kiosk/%d?access_token=%s' % (domain, record.id, record.access_token)
+
+    def action_close_kiosk_session(self):
+        current_session_id = self.current_session_id
+
+        if current_session_id:
+            if current_session_id.order_ids:
+                current_session_id.order_ids.filtered(lambda o: o.state != 'paid').unlink()
```

Comment:
We should be careful to this I think. In some localization, the orders are automatically invoiced so we would unlink paid orders. Maybe we should put
```suggestion
                current_session_id.order_ids.filtered(lambda o: o.state in ['cancel', 'draft']).unlink()
```

Reply:
I don't really understand the point